packages/html-reporter/src/testCaseView.spec.tsx

```diff
@@ -76,6 +76,13 @@ test('should render test case', async ({ mount }) => {
   await expect(component.getByText('My test')).toBeVisible();
 });
 
+test('should render copy buttons for annotations', async ({ mount }) => {
+  const component = await mount(<TestCaseView projectNames={['chromium', 'webkit']} test={testCase} run={0} anchor=''></TestCaseView>);
+  await expect(component.getByText('Annotation text', { exact: false }).first()).toBeVisible();
+  component.getByText('Annotation text', { exact: false }).first().hover();
+  await expect(component.getByRole('button').first()).toBeVisible();
```

Comment:
This feels very permissive. If there is any button in component, we think that there is a button and it works.

Reply:
Ah yeah I agree, I'll tighten this up to get the specific copy button and verify the copied text.